spyder/locale/spyder.pot

```diff
@@ -3524,7 +3524,7 @@ msgid "Unlock"
 msgstr ""
 
 #: spyder/plugins/help/plugin.py:645
-msgid "The following error occured when calling <b>Sphinx %s</b>. <br>Incompatible Sphinx version or doc string decoding failed.<br><br>Error message:<br>%s"
+msgid "The following error occurred when calling <b>Sphinx %s</b>. <br>Incompatible Sphinx version or doc string decoding failed.<br><br>Error message:<br>%s"
```

Comment:
Please revert this change. There's no need to touch pot files because they are updated internally after translations strings are updated (as you already did for this string in `spyder/plugins/help/plugin.py`).